bridge-fungible-token/tests/utils/environment.rs

```diff
@@ -100,12 +110,19 @@ abigen!(
         name = "ContractMessageScript",
         abi = "./bridge-message-predicates/contract_message_script-abi.json",
     ),
+    Contract(
+        name = "DepositRecipientContract",
+        abi =
+            "./test-deposit-recipient-contract/out/debug/test_deposit_recipient_contract-abi.json",
+    ),
 );
 
 pub const MESSAGE_SENDER_ADDRESS: &str =
     "0x00000000000000000000000096c53cd98B7297564716a8f2E1de2C83928Af2fe";
 pub const TEST_BRIDGE_FUNGIBLE_TOKEN_CONTRACT_BINARY: &str =
     "../bridge-fungible-token/out/debug/bridge_fungible_token.bin";
+pub const DEPOSIT_RECIPIENT_CONTRACT_BINARY: &str =
```

Comment:
Are these used outside of the file? If not then I would remove the `pub`. I'd use `pub(crate)` for any public things